pkg/kademlia/config.go

```diff
@@ -23,7 +23,7 @@ var (
 // Config defines all of the things that are needed to start up Kademlia
 // server endpoints (and not necessarily client code).
 type Config struct {
-	BootstrapAddr        string        `help:"the Kademlia node to bootstrap against" default:""`
+	BootstrapAddr        string        `help:"the Kademlia node to bootstrap against" default:"bootstrap.storj.io:8888"`
```

Comment:
we should totally not have the devDefault be this production value. the bootstrap node will get slammed with tests and storj-sims

Reply:
Agreed, do we have a reasonable dev default ?

Reply:
For Dev Default its best to leave it empty. Aka `releaseDefault:"bootstrap.storj.io:8888" devDefault:""`